source/_integrations/aftership.markdown

```diff
@@ -40,6 +40,11 @@ api_key:
   description: The API key for AfterShip.
   required: true
   type: string
+refresh_time:
```

Comment:
This is a bit of a weird name since this is called `scan_interval` in the Home Assistant world in general.